ui/app/components/transit-key-actions.js

```diff
@@ -35,213 +50,142 @@ const TRANSIT_PARAMS = {
   exportKeyType: null,
   exportKeyVersion: null,
   wrappedToken: null,
+  wrappedTTL: '30m',
```

Comment:
I set the default to "30m", which is what the WrapTtl sets it to farther down.

**Note**:` transit-key-actions/export.hbs` file is the only file that uses the `wrapTtl` component. Maybe it's something we can remove in another PR and replace the use case here with the TtlPicker instead?